cookbook/form/form_customization.rst

```diff
@@ -282,7 +282,7 @@ can now re-use the form customization across many templates:
 
 .. code-block:: html+jinja
 
-    {# app/Resources/views/Form/fields.html.twig #}
+    {# src/AppBundle/Resources/views/Form/fields.html.twig #}
```

Comment:
Anyhow. My point is that the path used in the code exampled throughout this page is inconsistent! I don't know what the best practice is. If you don't want to change it here, then please change it in the next codeblock (and probably some others).